doc/Install.md

```diff
@@ -145,6 +145,16 @@ Example:
 
 *Note*: Do **not** rename the .htaccess-dist file as it is tracked by GIT and renaming will cause a dirty working directory.
 
+### Verify the "host-meta" page is working
+
+Friendica should respond automatically to an important address named /.well-known/host-meta  This URL would look like, for example, https://example.com/.well-known/host-meta  It must be visible to the public and must respond with an XML file that is automatically customized to your site.
```

Comment:
It is not only ```.well-known/host-meta``` - there are more paths under ```.well-known```.

Reply:
I am not aware of them at all.  Could you help with this?

Reply:
There are several paths - that's why I would focus on the ```.well-known``` path.